build-tools/debian-metadata/oss-xamarin.android-targets.install

```diff
@@ -0,0 +1,11 @@
+bin/Release/bin/*					/usr/bin
```

Comment:
I don't disagree with `xabuild` being the only thing to go into `$PATH` (i.e. into `/usr/bin`)

The directory structure questions:
### FlatPak
`/app` is the only writable prefix for apps, and you put whatever you like in here. When you run an app, PATH is set to `/app/bin:/usr/bin` (`/usr` is provided by the runtime on which your app is built, in our case the minimal FreeDesktop runtime). In most cases, this just means stuff being built with `--prefix=/app` (e.g. `/app/bin/mono-sgen`). We could make `xabuild` the only thing in `/app/bin`, and XA would be installed anywhere (as long as it's a subdirectory in `/app`)
### Debian
The norm would be /usr/lib/packagename for platform-specific internal files, /usr/share/packagename for platform-independant internal files (which are really what we're talking about). If we're not shipping host-Linux (because it's only useful in the proprietary XS extension), then I don't think *any* of the files I noticed in the oss-xamarin.android bundle were platform specific - lots of cross-built Monos and lots of assemblies, but no binaries that would need to be rebuilt on amd64, i386, etc.

That said, I don't see a path to getting XA into distribution repositories (due to the hard dependency on an Internet connection during the build), and we can kinda do anything we like on mono-project.com packages which aren't beholden to packaging structure policy (i.e. `/opt/foo` is fine for non-distro packages)